CONTRIBUTING.md

```diff
@@ -1,20 +1,138 @@
-Thanks for wanting to contribute to Mozilla's Add-ons Linter! You rock! 😊
+Hi! Thanks for wanting to contribute to Mozilla's Add-ons Linter! You rock! 😊
 
-## Submitting a Pull Request
+This linter is used to help develop and publish [Add-ons](https://developer.mozilla.org/en-US/Add-ons/) for Firefox. You're an add-on developer, we would really value your contributions–no one knows add-on development and publishing better than an add-on developer!
```

Comment:
You may want to remove the `/en-US/` directory and let MDN redirect to the user's preferred locale.

Reply:
oh good point, damn copy & paste. Thanks!